libs/ledgerjs/packages/types-cryptoassets/src/index.ts

```diff
@@ -81,7 +81,6 @@ export type CryptoCurrencyId =
   | "particl"
   | "persistence"
   | "pirl"
-  | "pivx"
```

Comment:
I am wondering whether we should remove pivx in the ledgerjs package. because this package is used by other projects outside Ledger. This is why we see some LL unsupported currencies here. (e.g. monero)

Reply:
FYI they were removed for peercoin, stealthcoin, vertcoin, viacoin 
https://github.com/LedgerHQ/ledger-live/pull/7138/files#diff-8d5212f656c0023f496e2e16d624bfc345c426aa7f881b7b11b8dea43cf047ff
Was this a mistake also and do you think we should put them back in? Didn't seem to cause much trouble as no one complained but Im not 100% sure. In any case I'll put pivx back up 
You make a good point about monero and the package usage outside of ledger live.

Reply:
it is true that peercoin, stealthcoin, vertcoin, viacoin have done the same thing. Maybe we can sync with product to make a decision.